### PR TITLE
Fixes issue where `null` causes API calls to fail

### PR DESCRIPTION
**Pull request by djvirgen**

### CODE REVIEW

File: lib/Router.js [hunk 1]
```diff
@@ -423,7 +423,7 @@ var resolveSymlinks = function(results, connection, props){
             (function resolveSymlink(i) {
                 value = results[key];
 
-                if (value.hasOwnProperty('$link')) {
+                if (_.has(value, '$link')) {
```
Comment (skylargivens) [inline lib/Router.js:426]: Doesn't this have the same behavior as `hasOwnProperty`?

Comment (skylargivens) [inline lib/Router.js:426]: Oh, I see it now!

**Comment by skylargivens** [comment]
:+1:

**Comment by eviechan** [comment]
:cherry_blossom: :shipit:

**Pull request merged**